lib/webhook_system.rb

```diff
@@ -4,6 +4,7 @@
 require 'ph_model'
 require 'validate_url'
 require 'faraday'
+require 'webhook_system/faraday_middleware/encoding'
```

Comment:
let's put this file outside of the webhook_system namespace since its content doesn't match it either.